### ISSUE TITLE
Can't pass login screen

### ISSUE THREAD

**Issue by anadkatmeet** [widestage/widestage#55]
Couldn't login. When I pass administrator and widestage and login, it keeps loading. Nothing happens.
Note: I am using docker-compose

**Comment by ruben-s** [comment]
Hi, did you succeed in solving the issue?

I tried the same, docker-compose up, and a login page is available.
But I can't get past this login page.

Thx.

**Comment by beevou** [comment]
You sure that mongo db is up and running?

**Comment by ruben-s** [comment]
Hi Hermes,

yes mongodb is up an running (If I issue a "telnet 127.0.0.1 27017" then in
the docker console logs I see a connect).

But indeed the widestage docker instance reports an connection error:
ECONNREFUSED localhost:27017

So for some reason widestage/node.js fails to connect to mongodb.
I've played a bit with the docker-compose.yaml and the config.js files but
so far no luck.

I'll try again tonight if I find the time.

br,
 Ruben

**Comment by ruben-s** [comment]
Hi Hermes,
Hi Anadkatmeet,

I found a solution to the issue.
As Hermes indicated in my case widestage could not connect to mongodb.
It appears to have to do with the IP config.

When I run docker-compose up then 3 ip addresses are created: 172.18.0.2 to .4
(I found out by running iptables -L)
And as 172.18.0.3 is listening on 27017, I gathered this is mongodb.

I then changed the ip adress in the ./server/config/config.js in "production" section:
       db: 'mongodb://172.18.0.3:27017/widestage',

Then running docker-compose up again I can log in on 127.0.0.1 with administrator/widestage.

Question: how is the production section from the config.js selected?
How to switch to development or local?

Thx;

br,
 Ruben
